src/display/editor/editor.js

```diff
@@ -840,18 +840,16 @@ class AnnotationEditor {
     altText.tabIndex = "0";
     altText.addEventListener("contextmenu", noContextMenu);
     altText.addEventListener("pointerdown", event => event.stopPropagation());
-    altText.addEventListener(
-      "click",
-      event => {
-        event.preventDefault();
-        this._uiManager.editAltText(this);
-      },
-      { capture: true }
-    );
+
+    const onClick = (event, keyboard = false) => {
+      this.#altTextButton.hidden = true;
+      event.preventDefault();
+      this._uiManager.editAltText(this, /* isFromKeyboard */ keyboard);
```

Comment:
Can't we store the new state internally in the editor, e.g. as `this.#altTextIsFromKeyboard`, to avoid having to pass around this parameter to a whole bunch of different places?

Reply:
I wonder the same and I thought that it's useless to keep few bytes of memory for every editors when the real lifetime of interest is exactly the lifetime of the dialog, hence it should live in the dialog.

Reply:
Quite frankly I'm more "worried" about readability/maintainability of this code, and having to *needlessly* pass around this parameter doesn't really help as far as I'm concerned.

Reply:
I'm not sure to agree about readability/maintainability... but I trust you.